prepare.sh

```diff
@@ -68,7 +68,7 @@ export PYTHON_COMMAND=python3
 make -j"$(nproc)" -C BaseTools/
 {
     source ./edksetup.sh --reconfig
-    build -a X64 -b DEBUG -t GCC5 -DTPM2_ENABLE \
+    build -a X64 -b DEBUG -t GCC -DTPM2_ENABLE \
```

Comment:
Should we update also https://github.com/coconut-svsm/svsm/blob/main/Documentation/docs/installation/INSTALL.md#building-the-guest-firmware